src/libraries/Common/src/Extensions/ParameterDefaultValue/ParameterDefaultValue.netcoreapp.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
```

Comment:
Why are these `#nullable enable`s needed?

Reply:
Because the file is shared between 2 projects - one that has been annotated for nullable, and one that hasn't.

https://github.com/dotnet/runtime/blob/83e3c64793fcf3927045cd00c2c3a568bf2fa6ac/src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/Microsoft.Extensions.DependencyInjection.Abstractions.csproj#L6

https://github.com/dotnet/runtime/blob/83e3c64793fcf3927045cd00c2c3a568bf2fa6ac/src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj#L7